packages/patternfly-3/patternfly-react-extensions/src/components/CatalogTileView/__snapshots__/CatalogTileView.test.js.snap

```diff
@@ -0,0 +1,470 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CatalogTile renders properly 1`] = `
+<div
+  class="catalog-tile-view-pf"
+  id="test-catalog-tile-view"
+>
+  <div
+    class="catalog-tile-view-pf-category"
+    id="category-1"
+  >
+    <div
+      class="catalog-tile-view-pf-category-header"
+    >
+      <span
+        class="catalog-tile-view-pf-category-header-title"
+      >
+        Category 1
+      </span>
+      <span
+        class="pull-right"
+      >
+        <button
+          class="catalog-tile-view-pf-category-view-all pull-right btn btn-link"
+          style="margin-right: 0px;"
+          type="button"
+        >
+          View All (4)
+        </button>
+      </span>
+    </div>
+    <div
+      class="catalog-tile-view-pf-category-body container-fluid"
```

Comment:
not sure if this is the right place to leave this comment but why are you using `.container-fluid`? You're overwriting the `margin` and `padding`, so the only thing I think you get with this is the `:before/:after` clearfix stuff. If this needs a clearfix, you could just use `.clearfix`, then you could remove `padding: 0;` from `.catalog-tile-view-pf-category-body`